dashboards/rabbitmq/metadata.yaml

```
@@ -9,3 +9,12 @@ sample_dashboards:
     related_integrations:
       - id: rabbitmq
         platform: GCE
+  -
+    category: RabbitMQ
+    id: rabbitmq-prometheus-overview
+    display_name: RabbitMQ Prometheus Overview
+    description: |-
+      This dashboard is based on [built-in metrics](https://rabbitmq.com/prometheus.html). This dashboard has charts displaying several metrics related to Queues, Messages, and Nodes
+    related_integrations:
+      - id: rabbitmq
+        platform: GCE
```

I will update the contribution guide to reflect this, but we will use the platform code GKE for the prometheus-based integrations.

```suggestion
        platform: GKE
```